prover/lib/compressor/blob/blob.go

```diff
@@ -57,3 +48,34 @@ func GetDict() ([]byte, error) {
 	dictPath := filepath.Join(repoRoot, "prover/lib/compressor/compressor_dict.bin")
 	return os.ReadFile(dictPath)
 }
+
+func DecompressBlob(blob []byte, dictStore dictionary.Store) ([]byte, error) {
+	vsn := GetVersion(blob)
+	var (
+		blockDecoder func(*bytes.Reader) (encode.DecodedBlockData, error)
+		blocks       [][]byte
+		err          error
+	)
+	switch vsn {
+	case 0:
+		_, _, blocks, err = v0.DecompressBlob(blob, dictStore)
+		blockDecoder = v0.DecodeBlockFromUncompressed
+	case 1:
+		_, _, blocks, err = v1.DecompressBlob(blob, dictStore)
+		blockDecoder = v1.DecodeBlockFromUncompressed
+	default:
+		return nil, errors.New("unrecognized blob version")
+	}
+
+	if err != nil {
+		return nil, err
+	}
+	blockObjs := make([]encode.DecodedBlockData, len(blocks))
+	for i, block := range blocks {
+		blockObjs[i], err = blockDecoder(bytes.NewReader(block))
```

Comment:
Should you convert the decoded block to the standard representation from go-ethereum types.Block
```suggestion
		decodedBlock, err = blockDecoder(bytes.NewReader(block))
		blockObjs[i] = decodedBlock.ToStd()
```